### PR TITLE
Updated fr_CA locale

### PR DESCRIPTION
**Pull request by mhvezina**
@asmecher This update could also be integrated (copied) to branch `stable-3_1_2` if you deem this relevant.

Regarding issue https://github.com/pkp/pkp-lib/issues/5007, I have decided, while waiting for the fix in the English version, to base my translation on something like "This journal is collecting your ORCID iD so we can integrate your ORCID data with your user and author profile in OJS." as shown here: https://github.com/BibNumUMontreal/orcidProfile/blob/2d92217b9ef9dcf28a937d9e375e4d40eca97c21/locale/fr_CA/locale.xml#L61

### CODE REVIEW

File: locale/fr_CA/locale.xml [hunk 1]
```diff
@@ -9,28 +9,58 @@
...
+	<message key="plugins.generic.orcidProfile.instructions"><![CDATA[Vous pouvez préremplir ce formulaire avec les renseignements provenant d'un profil ORCID. Entrer l'adresse courriel ou l'identifiant ORCID associé au profil ORCID, puis cliquer sur « Envoyer ».]]></message>
+	<message key="plugins.generic.orcidProfile.noData">Aucun renseignement repéré sur ORCID</message>
+	<message key="plugins.generic.orcidProfile.emailOrOrcid">Adresse courriel ou identifiant ORCID :</message>
+	<message key="plugins.generic.orcidProfile.manager.settings.title">Paramètres de l'API ORCID</message>
+	<message key="plugins.generic.orcidProfile.manager.settings.hidden">caché-e</message>
+<message key="plugins.generic.orcidProfile.manager.settings.description">Veuillez configurer l'accès de l'API ORCID à utiliser afin extraire les renseignements du profil ORCID pour les importer dans le profil utilisateur ou celui de l'auteur-e et également mettre à jour les dossiers ORCID associés avec les données relatives aux nouvelles publications (uniquement pour les membres ORCID)</message>
```
Comment (asmecher) [inline locale/fr_CA/locale.xml]: Watch the indentation here! (Just for consistency's sake.)

**Comment by asmecher** [comment]
Merged, @mhvezina, and cherry-picked back to `stable-3_1_2` -- except for the email template change, which could not be cherry-picked automatically as there is a conflicting change between the two branches. Thanks for all the work!

File: locale/fr_CA/locale.xml (hunk 1 above)
Comment (mhvezina) [reply locale/fr_CA/locale.xml]: Sorry. I like it clean and consistent too!
Here it is : https://github.com/pkp/orcidProfile/pull/79

**Comment by mhvezina** [comment]
> Merged, @mhvezina, and cherry-picked back to `stable-3_1_2` -- except for the email template change, which could not be cherry-picked automatically as there is a conflicting change between the two branches. Thanks for all the work!

@asmecher I can do a PR with both fr_CA latest-consistent-indentation-locale.xml and emailTemplate.xml to the `stable-3_1_2` branch if you'd like?

**Comment by asmecher** [comment]
I've already done a cherry-pick for the indentation change (these are easy to do when there are no conflicting changes -- just mention in the PR that you'd like it cherry-picked too, and I'll save you the time). The email template change has a conflict, so I can't cherry-pick it directly -- so yes, a PR for that would be great!

**Pull request merged**